pkg/controllers/cassandra/nodepool/resource.go

```diff
@@ -49,7 +48,7 @@ func StatefulSetForCluster(
 			OwnerReferences: []metav1.OwnerReference{util.NewControllerRef(cluster)},
 		},
 		Spec: apps.StatefulSetSpec{
-			Replicas:    util.Int32Ptr(int32(np.Replicas)),
+			Replicas:    util.Int32Ptr(int32(0)),
```

Comment:
0?

Reply:
So that NodePools are always created with size 0 and then scaled out according to the logic in `ScaleOut.Execute`.

Reply:
Cool 👍